service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java

```diff
@@ -219,23 +209,8 @@ public void notifyDigitalRepresentatives(final AdditionalApplicationsUploadedEve
 
     private Set<String> getRepresentativesEmails(CaseData caseData,
                                                  RepresentativeServingPreferences servingPreference) {
-        AdditionalApplicationsBundle uploadedBundle = caseData.getAdditionalApplicationsBundle().get(0).getValue();
-
-        List<Element<Other>> othersSelected = getOthersSelected(uploadedBundle);
-        List<Element<Respondent>> respondentsSelected = getRespondentsSelected(uploadedBundle);
-
         Set<String> digitalRepresentatives = representativesInbox.getEmailsByPreference(caseData, servingPreference);
```

Comment:
does getEmailsByPreference include any others? (looks like they were getting filtered out from here previously if not selected)